.travis.yml

```diff
@@ -29,16 +29,17 @@ before_install:
     - source venv/bin/activate
     - python --version  # just to check
     - pip install -U pip wheel # upgrade to latest pip find 3.5 wheels; wheel to avoid errors
-    - retry pip install pytest pytest-cov pytest-dependency flake8 coverage codecov chardet "setuptools!=47.2.0" docutils check-manifest
+    - retry pip install flake8 coverage codecov chardet "setuptools!=47.2.0" docutils
```

Comment:
flake8 should also be in `dev`

you shouldn't need `coverage` since it's a dependency of `pytest-cov`

Reply:
We don't explicitly have anything which links to flake8 (aside from a few lines of config in setup.cfg), but I can add it in if you'd rather?

I'll drop coverage then.

Reply:
Done